cypress/integration/common/profile.js

```diff
@@ -28,9 +46,17 @@ When("I visit another user's profile page", () => {
   cy.openPage("profile/peter-pan");
 });
 
-Then("I cannot upload a picture", () => {
+Then("I cannot upload a profile picture", () => {
   cy.get(".base-card")
     .children()
     .should("not.have.id", "customdropzone")
     .should("have.class", "user-avatar");
 });
+
+
+Then("I cannot upload a profile header image", () => {
+  cy.get(".base-card")
+    .children()
+    .should("not.have.id", "profileHeaderDropzone")
+    .should("have.class", "profile-header-card");
```

Comment:
I suspect the fix you did changing `profile-header` to `profile-header-card` was wrong after all and the errors you thought it caused had in fact different causes. Since this test finds the element with `.base-card` class and then analyzes its children, its easy to see why it can't find `profile-header-card`. The test in its original state was in fact checking the child with the `profile-header` class:

![image](https://user-images.githubusercontent.com/26900172/85880152-d48db900-b7db-11ea-822e-a48ce299eb4c.png)


tldr; The error regarding this test should be fixed by changing line 61 back to 
```suggestion
    .should("have.class", "profile-header");
```